### ISSUE TITLE
Improper epytext in twisted.words.protocols.jabber.sasl_mechanisms

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8510]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8510|
|Type|enhancement|
|Created|2016-06-27 12:05:29Z|
|Branch|https://github.com/twisted/twisted/tree/jabber-typo-8510|

Improper epytext in twisted.words.protocols.jabber.sasl_mechanisms

<details><summary>Searchable metadata</summary>

```
trac-id__8510 8510
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__jabber_typo_8510 jabber-typo-8510
branch_author__acabhishek942 acabhishek942
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy_documentation easy documentation
time__1467029129603982 1467029129603982
changetime__1467042696857255 1467042696857255
version__None None
owner__ralphm ralphm

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See: https://github.com/twisted/twisted/pull/233

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_3"></a>|@ralphm set owner to @ralphm<br>@ralphm set status to `assigned`|
|-|-|

I pulled in this branch as `jabber-typo-85150` to have the bots do their thing.

This is a simple typo-change and everything checks out. Merging.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 4b3754826dfeca054b524c899dba6e65a3bfe1df
```
#!CommitTicketReference repository="" revision="4b3754826dfeca054b524c899dba6e65a3bfe1df"
Merge pull request #233 from acabhishek942/8510-acabhishek942-doc

Improper epytext removed from twisted.words.protocols.jabber.sasl_mechanisms

Author: acabhishek942
Reviewer: ralphm
Fixes: #8510
```
